core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1939,24 +1941,22 @@ private[spark] class DAGScheduler(
       hostToUnregisterOutputs: Option[String],
       maybeEpoch: Option[Long] = None): Unit = {
     val currentEpoch = maybeEpoch.getOrElse(mapOutputTracker.getEpoch)
+    logDebug(s"Removing executor $execId, fileLost: $fileLost, currentEpoch: $currentEpoch")
```

Comment:
Should this log goes into the if condition?

Reply:
See explanation in my response to your comment about the previous logDebug.
There are now two if clauses. I think it is simpler just to log the values at the beginning of the method; knowing the values will tell you which if clauses will get executed.